pkg/project/registry/project/proxy/proxy_test.go

```diff
@@ -100,7 +101,7 @@ func TestCreateProjectOK(t *testing.T) {
 
 func TestGetProjectOK(t *testing.T) {
 	mockClient := fake.NewSimpleClientset(&kapi.Namespace{ObjectMeta: kapi.ObjectMeta{Name: "foo"}})
-	storage := NewREST(mockClient.Core().Namespaces(), &mockLister{}, nil, nil)
+	storage := NewREST(mockClient.Core().Namespaces(), restclient.Config{}, &mockLister{}, nil, nil)
 	project, err := storage.Get(kapi.NewContext(), "foo")
 	if project == nil {
 		t.Error("Unexpected nil project")
```

Comment:
Can we have an explicit test for that new functionality here. The more tests, the better, especially when it comes to security.